tutorials/evaluation.md

```diff
@@ -9,59 +9,95 @@ You can evaluate LitGPT using [EleutherAI's lm-eval](https://github.com/Eleuther
 You need to install the `lm-eval` framework first:
 
 ```bash
-pip install 'lm_eval @ git+https://github.com/EleutherAI/lm-evaluation-harness.git@115206dc89dad67b8b'
+pip install lm_eval
 ```
 
 &nbsp;
 
 ### Evaluating LitGPT base models
 
-Use the following command to evaluate LitGPT models on all tasks in Eleuther AI's Evaluation Harness.
+Suppose you downloaded a base model that we want to evaluate. Here, we use the `microsoft/phi-2` model:
 
 ```bash
-python eval/lm_eval_harness.py \
-    --checkpoint_dir "checkpoints/meta-llama/Llama-2-7b-hf" \
-    --precision "bf16-true" \
-    --save_filepath "results.json"
+litgpt download --repo_id microsoft/phi-2
 ```
 
-To evaluate on LLMs on specific tasks, for example, TruthfulQA and HellaSwag, you can use the `--eval_task` flag as follows:
+The download command above will save the model to the `checkoints/microsoft/phi-2` directory, which we can
+specify in the following evaluation command:
 
-```bash
-python eval/lm_eval_harness.py \
-    --checkpoint_dir "checkpoints/meta-llama/Llama-2-7b-hf" \
-    --eval_tasks "[truthfulqa_mc,hellaswag]" \
-    --precision "bf16-true" \
-    --save_filepath "results.json"
+
+```
+litgpt evaluate \
+  --checkpoint_dir checkpoints/microsoft/phi-2/ \
+  --batch_size 4 \
+  --out_dir evaluate_model/
 ```
 
-A list of supported tasks can be found [here](https://github.com/EleutherAI/lm-evaluation-harness/blob/master/docs/task_table.md).
+The resulting output is as follows:
 
-&nbsp;
+```
+...
+|---------------------------------------|-------|------|-----:|--------|-----:|---|-----:|
+...
+|truthfulqa_mc2                         |      2|none  |     0|acc     |0.4656|±  |0.0164|
+|hellaswag                              |      1|none  |     0|acc     |0.2569|±  |0.0044|
+|                                       |       |none  |     0|acc_norm|0.2632|±  |0.0044|
+
+|      Groups      |Version|Filter|n-shot|Metric|Value |   |Stderr|
+|------------------|-------|------|-----:|------|-----:|---|-----:|
+|mmlu              |N/A    |none  |     0|acc   |0.2434|±  |0.0036|
+| - humanities     |N/A    |none  |     0|acc   |0.2578|±  |0.0064|
+| - other          |N/A    |none  |     0|acc   |0.2401|±  |0.0077|
+| - social_sciences|N/A    |none  |     0|acc   |0.2301|±  |0.0076|
+| - stem           |N/A    |none  |     0|acc   |0.2382|±  |0.0076|
+```
 
-### Evaluating LoRA-finetuned LLMs
 
-The above command can be used to evaluate models that are saved via a single checkpoint file. This includes downloaded checkpoints and base models finetuned via the full and adapter finetuning scripts.
+Please note that the `litgpt evaluate` command run an internal model conversion. 
+This is only necessary the first time you want to evaluate a model. To skip the conversion, 
+when you want to evaluate a model a second time, you can pass the `--skip_conversion true` argument:
```

Comment:
This says skip_conversion but the code implements force_conversion

Reply:
Ah yes, this was changed based on Adrian's suggestion. We know don't run the conversion redundantly anymore. Users don't really need to know about `--force_conversion` in most cases, so I removed that from the docs.

Reply:
Should we remove the argument from the script too then?

Reply:
I find it useful to have as an option. E.g. if your model changed and you want to evaluate the updated model you can force the conversion. The alternative is to manually delete the files.